flask_rebar/rebar.py

```diff
@@ -83,36 +81,30 @@ def wrapped(*args, **kwargs):
 
         rv = f(*args, **kwargs)
 
-        if marshal_schema:
-            if isinstance(rv, tuple):
-                data, status_code = rv[0], rv[1]
```

Comment:
The existing behavior ignores any headers the user may have passed in the third element of the tuple, and also doesn't support passing a 2-tuple like `(body, headers)`, or a string status (see "tuple" under http://flask.pocoo.org/docs/1.0/api/#flask.Flask.make_response)